php.md

```diff
@@ -0,0 +1,43 @@
+# PHP
+
+
+## Prerequisite knowledge:   
+HTML
+
+
+## Classification / Acceptance :
+
+#### Apprentice aka _Hello World_
+ * what kind of language PHP is?
+ * explain the process of generating HTML code ‘by’ PHP script
```

Comment:
Quoting https://github.com/bethinkpl/bit-docs/blob/master/!document_template.md:
>Questions students could ask themeselves to evaluate their competence level.

Please reformulate all the points to questions. You can take a look at other docs for inspiration.